conferences.ics

```diff
@@ -17,4 +19,6 @@ DTSTART;VALUE=DATE:{{ conference.date_start | date: "%Y%m%d" }}
 DTEND;VALUE=DATE:{{ conference.date_end | date: "%s" | plus: 86400 | date: "%Y%m%d" }}
 DTSTAMP:{{ site.time | date: "%Y%m%dT%H%M%S" }}Z
 END:VEVENT
-{% endfor %}END:VCALENDAR
+{%- endif -%}
```

Comment:
You're now introducing `\n`s here. Have you checked that the iCal works correctly?

Reply:
The hyphens at the start and/or end of the liquid tags lead to unwanted whitespace (including line breaks) being stripped. See https://shopify.dev/docs/themes/liquid/reference/basics/whitespace

Works on my machine™